src/librustc_typeck/check/method/suggest.rs

```diff
@@ -284,7 +285,9 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                                               rcvr_expr,
                                               out_of_scope_traits);
 
-                if let Some(lev_candidate) = lev_candidate {
+                if let Some(whitelist_candidate) = whitelist_candidate {
+                    err.help(&format!("did you mean `{}`?", whitelist_candidate.name));
```

Comment:
Given that this suggestion is coming from a whitelist it'd mean that you'd be very confident with it, right? In that case you can use `err.span_suggestion(sp, msg, new_code)`.

Reply:
Ah, thanks!